recipes/prismatic/build.sh

```diff
@@ -0,0 +1,11 @@
+
+
+mkdir build && cd build 
+
+cmake -D PRISMATIC_ENABLE_GUI=1 \
+	-D CMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
```suggestion
	-D CMAKE_INSTALL_PREFIX=$PREFIX \
	-DCMAKE_PREFIX_PATH=${PREFIX} \
```

Reply:
I'm unclear what this change does - is it standardising whitespace?

Reply:
I've added the CMAKE_PREFIX_PATH - I'll test later if this helps with finding the cdt packages when they're moved from build to host.